assets/xml/objects/object_bee.xml

```diff
@@ -1,19 +1,19 @@
 ﻿<Root>
     <File Name="object_bee" Segment="6">
-        <Animation Name="object_bee_Anim_00005C" Offset="0x5C" />
-        <DList Name="object_bee_DL_000550" Offset="0x550" />
-        <DList Name="object_bee_DL_000780" Offset="0x780" />
-        <DList Name="object_bee_DL_000848" Offset="0x848" />
-        <DList Name="object_bee_DL_000918" Offset="0x918" />
-        <DList Name="object_bee_DL_0009D0" Offset="0x9D0" />
+        <Animation Name="gBeeIdleAnim" Offset="0x5C" />
```

Comment:
Since there is only one animation for both hostile/idle, I think "flying" could be an option to cover both cases.

Reply:
good point 👍